Reflection_Engine/Query/Description.cs

```diff
@@ -147,6 +148,14 @@ public static string Description(this Type type, ClassificationAttribute classif
                 return desc;
             }
 
+            // If this is a Quantity, return the description of the QuantityAttribute
```

Comment:
Also, this is not picked up for a collection of quantities. (Like a List<Quantity<Length>). THis is due to the filtering on line 63 in this file that only checks if the property is of type `IObject`, but does not check for List<IObject>.